danielMunteanu/src/main/java/danielCollectionTask/Item.java

```diff
@@ -0,0 +1,38 @@
+package danielCollectionTask;
+
+public class Item {
+    public long id;
+    String title;
```

Comment:
any access modifiers

Reply:
fixed in next commit

Reply:
please get rid of such comment, it's a bit confusing in which commit changes have been made, e.g. Done, Fixed, Updated